frontend/public/touchpoints.js

```diff
@@ -822,4 +821,4 @@ const touchpointsFormHtmlString = `<div id="fba-modal-dialog" class="fba-modal-d
 </div>
 `;
 
-const touchpointForm5b1efe87 = new FBAform(document, window).init(formOptions);
+new FBAform(document, window).init(formOptions);
```

Comment:
Looks like LGTM is annoyed about this `window` param - I think it can be safely removed.